### PR TITLE
Desktop + Mobile layout changes

### PR DESCRIPTION
**Pull request by groeneman**
Most of Sarah's design changes implemented here. I'm not particularly strong on CSS, so there are a few differences between Sarah's design and what appears here. Most notably, I couldn't figure out how to cleanly shift the "P" logo to the left of the header text without reworking a ton of HTML. If it's very important, perhaps I can look at that as a follow-on PR. I'm sure there are other details I didn't get quite right, happy to give them another look but figured it was better to get the basic changes out without delay.

Dektop (before/after)

Screen Shot 2020-08-23 at 10 14 08 PM

Screen Shot 2020-08-23 at 10 13 45 PM

Mobile (before/after)

www powerthepolls org_(iPhone 6_7_8)

localhost_3333_(iPhone 6_7_8)

### CODE REVIEW

**Review by mojowen — DISMISSED** [review]

File: site/src/components/page-form/page-form.tsx [hunk 1]
```diff
@@ -39,10 +39,6 @@ export class PageForm {
          ? ( PartnerList.filter( p => p.partnerId === partnerId ) || [null] )[0]
          : null;
 
-      let daysLeft = Math.round( ( new Date( 2020, 9, 1 ).getTime() - Date.now() ) / 1000 / 60 / 60 / 24 / 7 ) * 7;
-      // count down every 10 days (since the 2020-10-01 end is arbitrary) by extracting off the days less than 10 and rounding up or down
-      daysLeft = ( daysLeft - daysLeft % 10 ) + Math.round( daysLeft % 10 / 10 ) * 10;
-
       const formCompleted = () => {
```
Comment (nexussays) [inline site/src/components/page-form/page-form.tsx:42]: My super cool algorithm :'( so sad

**Review by nexussays — CHANGES_REQUESTED** [review]
Looks good, thanks! Only thing is if you could remove the `on-desktop` mixin so that the default is always consistent and `on-mobile` remains the only modifier.

**Comment by nexussays** [comment]
Also @groeneman if you don't mind merging in from `master` just to make it a little cleaner since a few branches have since been merged to master.

**Comment by groeneman** [comment]
> Looks good, thanks! Only thing is if you could remove the `on-desktop` mixin so that the default is always consistent and `on-mobile` remains the only modifier.

@nexussays think I'm confused on this. Our subhead text is totally different on desktop and mobile, and so I think I need both mixins to hide each one on their respective screen sizes. Neither can be shown "by default"

**Review by nexussays — CHANGES_REQUESTED** [review]
@groeneman any reason the logo image changed? I don't think it should have and the new one is huge (1.5MiB)

**Comment by nexussays** [comment]
> 
> 
> > Looks good, thanks! Only thing is if you could remove the `on-desktop` mixin so that the default is always consistent and `on-mobile` remains the only modifier.
> 
> @nexussays think I'm confused on this. Our subhead text is totally different on desktop and mobile, and so I think I need both mixins to hide each one on their respective screen sizes. Neither can be shown "by default"

The styles without the mixin are the default. In order to keep that consistent, we treat large screens as the default and make an exception for small screens using `on-mobile`. So if something is hidden on large screens then set `display: none` and `@on-mobile{ display: block; }`, and so forth for padding or whatever else.

**Comment by groeneman** [comment]
@nexussays re: logo, Sarah specifically requested using a new version of the logo (I think there were some compression effects or a jagged edge in the old one). I did forget to reduce the new file down to an appropriate size, so I just cut a smaller version and committed that along with the source file.

**Comment by groeneman** [comment]
@nexussays got it, removed `on-desktop`, ready for review again :)

**Review by nexussays — APPROVED** [review]

**Pull request merged**